tv_apps/tv-ftu/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1">
+    <meta charset="utf-8">
+    <title>FTU - App Usage Metrics</title>
+
+    <!-- stylesheets of building blocks -->
+    <link rel="stylesheet" type="text/css" href="bower_components/smart-icons/smart-icons.css">
+    <link rel="stylesheet" type="text/css" href="bower_components/smart-button/style.css">
+    <link rel="stylesheet" type="text/css" href="style/index.css">
+
+    <!-- building block -->
+    <script defer src="bower_components/evt/index.js"></script>
+    <script defer src="bower_components/smart-button/script.js"></script>
+
+    <!-- For test purpose on phone only -->
+    <script defer src="shared/js/gesture_detector.js"></script>
+    <script defer src="shared/js/smart-screen/init_gesture.js"></script>
+
+    <!-- shared libraries -->
+    <script defer src="shared/js/settings_listener.js"></script>
+    <script defer src="shared/js/smart-screen/simple_key_navigation.js"></script>
+    <script defer src="shared/js/smart-screen/key_navigation_adapter.js"></script>
+
+    <!-- main javascript code -->
+    <script defer src="js/index.js"></script>
+
+    <!-- Shared L10n libraries -->
+    <script src="shared/js/l10n.js"></script>
+
+    <!-- L10n properties -->
+    <meta name="defaultLanguage" content="en-US">
+    <meta name="availableLanguages" content="en-US">
+    <link rel="localization" href="locales/ftu.{locale}.properties"/>
+  </head>
+
+  <body>
+
+    <article id="appUsageSlide" class="slide">
+      <section class="slide-section top">
+        <img id="appUsageSlide-logo" src="style/images/firefox_logo.png">
+      </section>
+      <section id="appUsageSlide-termsSection" class="slide-section mid" tabindex="-1">
+        <div id="appUsageSlide-terms">
+          <p id="appUsageSlide-terms-title" class="slide-description txt-large txt-center" data-l10n-id="about-firefox-os"></p>
+          <p class="slide-description txt-mid" data-l10n-id="welcome-to-fx-os"></p>
```

Comment:
can you use more meaningful naming for css like -title -content -panel rather than -top -mid -botttom?